act2.sh

```diff
@@ -1,13 +1,14 @@
 # 1. Inside the `star_wars/rebellion` directory, (IN ONE COMMAND) create a file called princess_leia.txt with the text "Help me, Obi-Wan…You’re my only hope."
-
+touch princess_leia.txt && echo "Help me, Obi-Wan…You’re my only hope." >> princess_leia.txt
 # 2. Create a file called obi_wan.txt in star_wars/rebellion
-
+touch obi_wan.txt
 # 3. Create a file in star_wars/rebellion called luke_skywalker.txt
-
+touch luke_skywalker.txt
 # 4. Create a directory in star_wars/rebellion called millenium_falcon
-
+mkdir millenium_falcon
 # 5. Inside the millenium_falcon, create two files: han_solo.txt and chewbacca.txt
-
+touch han_solo.txt chewbacca.txt
 # 6. Move luke_skywalker, obi_wan, and princess_leia into the millenium_falcon, respectively.
-
+ mv ~/sei/homework/Star_Wars/rebellion/luke_skywalker.txt ~/sei/homework/Star_Wars/rebellion/millenium_falcon/
```

Comment:
Looks like a direct path? This is fine here because it is not too nested, but a relative path would be more helpful! It all depends on the directory you're currently in! :)

Reply:
Thank you for the insight! I have to say I was pretty terrified rolling in to class Monday. You and the team have been so supportive it has set my mind at ease. Cannot thank you enough